python/ray/train/v2/_internal/execution/controller/controller.py

```diff
@@ -484,6 +484,10 @@ def _generate_run_attempt_id(self):
     def _get_run_attempt_id(self):
         return self._run_attempt_id
 
+    def ping(self) -> bool:
```

Comment:
I think you can just use is_actor_alive (https://github.com/ray-project/ray/blob/master/python/ray/train/v2/_internal/state/state_actor.py#L101). Otherwise the client needs to wait until the controller async awaits before getting a response.